clang/www/get_started.html

```diff
@@ -74,8 +74,8 @@ <h3 id="buildNix">On Unix-like Systems</h3>
         <a href="https://llvm.org/docs/CMake.html#frequently-used-cmake-variables">frequently used cmake variables</a>
         for more options.
     </li>
-    <li><tt>cmake -DLLVM_ENABLE_PROJECTS=clang -DCMAKE_BUILD_TYPE=Release -G "Unix Makefiles" ../llvm</tt></li>
```

Comment:
Sorry, I don't think we can tell people to use ninja here as a matter of fact, because it's not in our requirements documented here: https://llvm.org/docs/GettingStarted.html#software. We still can suggest it somewhere if it's available.

Reply:
Oh I thought someone told me to use Ninja here in the docs I can revert. It's already referenced for Windows though - https://github.com/llvm/llvm-project/blob/12b0ef5dade664fb24cb4e627f1c391d937c0925/clang/www/get_started.html#L216 - still want the revert?

Reply:
> Oh I thought someone told me to use Ninja here in the docs I can revert.

That was me, sorry for that.

> It's already referenced for Windows though

Oh, I never noticed. It's fine there, because above in "Using Ninja alongside Visual Studio" an explanation is given about it. We can keep `-GNinja` (which is my preference), if we can generalize that section for both Windows and Linux.